docs/pages/admin-guides/management/guides/migrating-iam-ic-from-okta-to-teleport.mdx

```diff
@@ -0,0 +1,359 @@
+---
+title: Migrating AWS Identity Center from Okta to Teleport
+description: A guide for gradually migrating an Identity Center instance from Okta control to Teleport control
+tocDepth: 3
+---
+
+The goal of this guide is to allow you to perform a staged migration from
+Okta-managed AWS Identity Center users & groups, to having them managed by Teleport
+while preserving Okta as your AWS SSO provider.
+
+This approach lets you get set up with a minimal Teleport AWS Identity Center
+integration that you can expand as you gain expertise and confidence with
+Teleport, as well as offering deployment options that are not available with a 
+"standard" Teleport Identity Center configuration, such as having Teleport deployed
+in a private network.
+
+You can also use this setup if you don't want to migrate fully to using Teleport
+as an Identity Provider for AWS Identity Center. Following the steps in this
+guide will allow you to have Teleport control user and group AWS account
+Assignments, while preserving Okta as AWS Identity Center's IdP.
+
+## How it works
+
+By default, the Teleport AWS Identity Center integration:
+- controls all user and group provisioning into Identity Center, and 
+- becomes the SSO provider for AWS. 
+
+This is fine for most deployments, but can cause difficulties when deploying into
+a sophisticated, Okta-managed AWS Identity Center configuration. 
+
+By contrast, the deployment in this guide will transition transition control
+from Okta to Teleport in a staged fashion.
+
+### Migration Path
+
+#### Starting Point
+
+This is our starting configuration: Okta as sole Identity Source for Identity Center.
+
+![Migration Starting Point](../../../../img/identity-center/ic-migrate-start.png)
+
+#### Midway
+
+This is the point where you can start migrating control of users and groups from
+Okta to Teleport. At this stage:
+
+ - Okta provides SSO login for Identity Center
+ - Okta manages a subset of the Identity Center group membership (selected via Push Groups)
+ - Okta and Teleport both control user provisioning via SCIM
+ - Teleport manages a second subset of Identity Center groups (selected via group filters during plugin installation)
+ - Teleport controls IC Group Account Assignments for the Identity Center groups under its control
+ - Teleport controls direct IC User Account Assignments for the Identity Center users under its control
+
+![Migration Mid Point](../../../../img/identity-center/ic-migrate-mid.png)
+
+#### Ending point
+
+Once Teleport is in control of all the users and groups you want to provision
+into Identity Center, the Okta Provisioning and Push Groups functions can be
+retired, leaving Okta only providing SSO login.
+
+![Migration End Point](../../../../img/identity-center/ic-migrate-end.png)
+
+## Prerequisites
+
+ - A working Teleport cluster
+ - An AWS role configured as per the [Identity Center guide](./aws-iam-identity-center.mdx)
```

Comment:
I think we should scope down the sub-heading link so reader can dive straight into it instead of reading all of that doc. E.g. `(./awsxxxx.mdx#sub-heading)`